plugins/kafka-info/src/components/KafkaInfoComponent/KafkaInfoComponent.tsx

```diff
@@ -111,7 +209,12 @@ export function KafkaInfoComponent() {
   };
 
   return (
-    <InfoCard title={title} noPadding>
+    <InfoCard title={title}>
```

Comment:
We talked about unifying these `<InfoCard/>` components so that you don't need to recreate it multiple times. That's still a viable option. I wouldn't die on this hill, it isn't a ton of lines being duplicated, but it is some, and there's some differences, see the `<Grid/>` components in `if (loading)` - I suggest taking a pass at it.